components/SortPlants/index.js

```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+import { plants as plantsData } from "@/lib/data";
```

Comment:
Currently it works to import the plantData when you need it. I would prefer to hand them over as props to this component as it might then be easier to refactor the code later when adding the database connection (the moment where you will need to work with props in this case). :)